RTPacket.cs

```diff
@@ -1250,6 +1249,22 @@ public List<Camera> Get2DMarkerData()
             }
         }
 
+        /// <summary>
+        /// Get 2D marker data
+        /// </summary>
+        public void Get2DMarkerData(List<Camera> output)
+        {
+            output.Clear();
+            lock (packetLock)
+            {
+                int c = m2DMarkerData.Count;
+                for (int i = 0; i < c; ++i)

```

Comment:
Cant we use foreach instead? Since count (c) is not being used

Reply:
Thanks for the feedback. I've benchmarked this and came to the conclusion it was not worth using foreach since it's abit slower. I don't think it hurts readability too much either.